h2o-r/h2o-package/R/frame.R

```diff
@@ -38,6 +38,8 @@
 #-----------------------------------------------------------------------------------------------------------------------
 # Private/Internal Functions
 #-----------------------------------------------------------------------------------------------------------------------
+#library(slam)
+#library(data.table)
```

Comment:
Loading the libraries this way should not be necessary.  Delete these two lines (if that causes problems, let me know).